Project-II-Calculator/calculator/src/App.js

```diff
@@ -1,22 +1,48 @@
-import React from 'react';
-import './App.css';
+import React from "react";
+import "./App.css";
+
+import NumberButton from "./components/ButtonComponents/NumberButton";
+import ActionButton from "./components/ButtonComponents/ActionButton";
+import CalculatorDisplay from "./components/DisplayComponents/CalculatorDisplay";
+
+// import ButtonContainer from './components/ButtonComponents/ButtonContainer';
 
 const App = () => {
   return (
-    <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
-    </div>
+    <div className="wrapper">
+      <CalculatorDisplay />
+
+      <div className="bcontainer">
+        <ActionButton style="wide clear" text="clear" />
+        <ActionButton style="operation" text="/" />
+      </div>
+
+      <div className="bcontainer">
+        <NumberButton style="num" text="7" />
+        <NumberButton style="num" text="8" />
+        <NumberButton style="num" text="9" />
+        <ActionButton style="operation" text="X" />
+      </div>
+
+      <div className="bcontainer">
+        <NumberButton style="num" text="4" />
+        <NumberButton style="num" text="5" />
+        <NumberButton style="num" text="6" />
+        <ActionButton style="operation" text="-" />
+      </div>
+
+      <div className="bcontainer">
+        <NumberButton style="num" text="1" />
+        <NumberButton style="num" text="2" />
+        <NumberButton style="num" text="3" />
+        <ActionButton style="operation" text="+" />
+      </div>
+
+      <div className="bcontainer">
+        <ActionButton style="wide zero" text="0" />
+        <ActionButton style="operation" text="=" />
```

Comment:
This is a bit tricky. Apparently, you were able to use the word "style" as a prop key and it worked fine. I still don't understand why, because "style" is a reserved keyword used to add inline-styles to React components. But if you looked at the compiler in your terminal, it was probably yelling at you. You can just change it to something like this, and it will stop yelling at you:
```suggestion
        <ActionButton styling="operation" text="=" />
```
Note that you'll have to go through and change everywhere you tried to use the "style" prop key.

Here's some more info:

https://blog.logrocket.com/the-best-styling-in-react-tutorial-youve-ever-seen-676f1284b945

Reply:
Other than this, the calculator looks great.